benchmarks/OptimizationFrameworks/optimal_powerflow.md

```diff
@@ -0,0 +1,1767 @@
+---
+title: Optimal Powerflow Nonlinear Optimization Benchmark
+author: Chris Rackauckas
+---
+
+## Optimization.jl
+
+Constraint optimization implementation reference: https://github.com/SciML/Optimization.jl/blob/master/lib/OptimizationMOI/test/runtests.jl
+Other AD libraries can be considered: https://docs.sciml.ai/dev/modules/Optimization/API/optimization_function/
+However ForwardDiff is the only one that is compatible with constraint functions
```

Comment:
Except Tracker all other backends now support constraints

Reply:
Yes, let's get this working and then merged. Then after merging, let's setup the benchmark to run with multiple Optimization.jl choices.